holonote/tests/test_annotation_table.py

```diff
@@ -1,26 +1,100 @@
 from __future__ import annotations
 
+import numpy as np
 import pandas as pd
 
 from holonote.annotate import AnnotationTable
 
 
-def test_table_region_df():
+def test_table_single_kdim() -> None:
```

Comment:
What is the point of typing test methods? They will all return `None`!

Reply:
Mostly, preparation for when a type checking is run as part of the CI.